_blog-src/_posts/2015-07-07-fuse-apis.markdown

```diff
@@ -0,0 +1,171 @@
+---
+layout: post
+title:  "Manage Fuse APIs with apiman"
+date:   2015-07-07 15:15:15
+author: Stefan Bunciak
```

Comment:
This should be sbunciak@redhat.com

Go to `_blog_src/_data/authors.yml` and make your entry there :-)